apps/OboeTester/app/src/main/java/com/mobileer/oboetester/OboeAudioStream.java

```diff
@@ -258,6 +264,15 @@ public int getSessionId() {
     }
     private native int getSessionId(int streamIndex);
 
+    public String getPackageName() {
+        return getPackageName(streamIndex);
```

Comment:
It is not related to this PR, but why `streamIndex` doesn't have prefix `m`.

I'm a little bit confused when I set the function here call `getPackageName(streamIndex)` while `streamIndex` is the parameter name for the bottom native method. And finally figure out `streamIndex` is a local member for the class.